tests/test_convert.py

```diff
@@ -41,6 +41,7 @@ def test_conversions(tmp_path: Path, package_name, branch):
     dist_git_path.mkdir(parents=True)
     sg_path.mkdir(parents=True)
     convert_repo(package_name, dist_git_path, sg_path, branch=branch)
+    os.chdir(sg_path)
```

Comment:
Is this here for a purpose or just some testing?

Reply:
```
test_conversions: chdir to source-git before running packit

because pytest can't write the SRPM to /
```
see previous test results

Reply:
Ok, so what about mounting the sources to `/src` and set container working-dir there so we don't need to do this each time we need to write anything?